robot/web/stream/server.go

```diff
@@ -9,120 +11,62 @@ import (
 	"github.com/pion/webrtc/v3"
 	"go.uber.org/multierr"
 	streampb "go.viam.com/api/stream/v1"
+	"go.viam.com/rdk/gostream"
 	"go.viam.com/utils"
 	"go.viam.com/utils/rpc"
 )
 
-// StreamAlreadyRegisteredError indicates that a stream has a name that is already registered on
-// the stream server.
-type StreamAlreadyRegisteredError struct {
-	name string
-}
-
-func (e *StreamAlreadyRegisteredError) Error() string {
-	return fmt.Sprintf("stream %q already registered", e.name)
-}
-
-// A StreamServer manages a collection of streams. Streams can be
-// added over time for future new connections.
-type StreamServer interface {
-	// ServiceServer returns a service server for gRPC.
-	ServiceServer() streampb.StreamServiceServer
-
-	// NewStream creates a new stream from config and adds it for new connections to see.
-	// Returns the added stream if it is successfully added to the server.
-	NewStream(config StreamConfig) (Stream, error)
-
-	// AddStream adds the given stream for new connections to see.
-	AddStream(stream Stream) error
+type Server struct {
```

Comment:
As per Go's conventions, I changed `StreamServer` -> `Server`. Because "Stream" is already a part of the package name.

More importantly, I got rid of the interface and just returned the bare structure type.

The only consequence of removing the interface was that the "standalone server" relied on it. Thus me just deleting that file.